assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -203,6 +215,13 @@ function calculatePageThumbSize(carouselSize) {
   return [pageWidth + THUMB_FRAME_WIDTH, pageHeight + THUMB_FRAME_HEIGHT];
 }
 
+function useCanvasSelector({
```

Comment:
Functions starting with `use*` are expected to be hooks, but this is not. I understand the naming here, but perhaps `safeZoneSelector` would be more appropriate.